frontend/src/components/filterable-table.js

```diff
@@ -49,18 +52,21 @@ function FilterableTable(props) {
         setSelected([..._selected]);
     };
 
-    const [filterString, setFilterString] = React.useState("");
+    const [filterStrings, setFilterStrings] = React.useState([]);
     const [lastSelected, setLastSelected] = React.useState(null);
     const [allSelected, setAllSelected] = React.useState(false);
     function isSelected(id) {
         return _selected.has(id);
     }
 
-    const filteredData = filterString
-        ? data.filter((row) =>
-              rowToStr(row).includes(filterString.toLowerCase())
-          )
-        : data;
+    const filteredData =
+        filterStrings.length === 0
+            ? data
+            : filterStrings.reduce((filteredData, filterString) => {
```

Comment:
This is confusing to me...Use something like `rows.filter(row => filterStrings.every(string => row.match(string)))`

Reply:
Replaced with :
~~~
 const filteredData = data.filter((row) =>
        filterStrings.every((string) => rowToStr(row).match(string))
    );
~~~
Much better!

Reply:
let's optimize a little bit and only convert the row to a string once for each item.